### DISCUSSION TITLE
Awesome Livewire

### DISCUSSION THREAD

**Discussion by amaelftah** [livewire/livewire#971] [4. General]
Since we have [awesome-alpine](https://github.com/alpinejs/awesome-alpine) what about to do the same for livewire

**Comment by imliam** [comment]
I've just quickly put this together: https://github.com/imliam/awesome-livewire

Happy to take PRs and transfer it to the Livewire organisation if Caleb wants that.

**Reply by amaelftah** [reply]
thanks a lot @imliam
